tests/unit/api/objectPutACL.js

```diff
@@ -461,4 +461,58 @@ describe('putObjectACL API', () => {
                 });
         });
     });
+
+    it('should set originOp to s3:ObjectAcl:Put when ACL is changed', done => {
+        const testObjACLRequest = {
+            bucketName,
+            namespace,
+            objectKey: objectName,
+            headers: { 'x-amz-acl': 'public-read-write' },
+            url: `/${bucketName}/${objectName}?acl`,
+            query: { acl: '' },
```

Comment:
we may want to run this test with different kind of updates : i.e. testing all paths of the condition on line 22 in acl.js :
```
if (cannedChanged || !_didNotChange('FULL_CONTROL') || !_didNotChange('WRITE_ACP')
            || !_didNotChange('READ') || !_didNotChange('READ_ACP')) 
```